logs/all.log

```diff
@@ -0,0 +1,30 @@
+2022-08-10 15:04:37,026 DEBUG m.j.c.s.o.C.Cities [main] Some debug log
```

Comment:
logs should be in .gitignore
.idea folder should be in .gitignore
out folder should be in .gitignore